eradiate/solvers/core/_solver_app.py

```diff
@@ -156,9 +156,9 @@ def postprocess(self, measure: Optional[Measure] = None):
 
         # Collect illumination angular data
         illumination = self.scene.illumination
-        saa = illumination.azimuth.m_as(ureg.rad)
-        sza = illumination.zenith.m_as(ureg.rad)
-        cos_sza = np.cos(sza)
+        saa = illumination.azimuth.m_as(ureg.deg)
+        sza = illumination.zenith.m_as(ureg.deg)
+        cos_sza = np.cos(illumination.zenith.m_as(ureg.rad))
```

Comment:
Why change the units used for post-processing?

Reply:
I reverted it.